packages/jsii-dotnet-runtime-test/test/Amazon.JSII.Runtime.IntegrationTests/ComplianceTests.cs

```diff
@@ -1197,13 +1239,11 @@ public SubclassNativeFriendlyRandom()
                 _nextNumber = 100;
             }
 
-            [JsiiMethod(name: "hello", returnsJson: "{\"type\":{\"primitive\":\"string\"}}", isOverride: true)]
```

Comment:
I am amazed that this was how these tests were written... Did we really expect users to annotate all their methods like this??

Reply:
🤷🏻‍♂️ but to be honest figuring out how to get annotations from interfaces was... a challenge. But I do agree. In terms of ergonomics it was... _terrible_.